Mage.Sets/src/mage/cards/s/StrefanMaurerProgenitor.java

```diff
@@ -0,0 +1,174 @@
+package mage.cards.s;
+
+import mage.MageInt;
+import mage.abilities.Ability;
+import mage.abilities.common.AttacksTriggeredAbility;
+import mage.abilities.common.BeginningOfEndStepTriggeredAbility;
+import mage.abilities.costs.common.SacrificeTargetCost;
+import mage.abilities.dynamicvalue.DynamicValue;
+import mage.abilities.effects.ContinuousEffect;
+import mage.abilities.effects.Effect;
+import mage.abilities.effects.OneShotEffect;
+import mage.abilities.effects.common.CreateTokenEffect;
+import mage.abilities.effects.common.DoIfCostPaid;
+import mage.abilities.effects.common.continuous.GainAbilityTargetEffect;
+import mage.abilities.hint.Hint;
+import mage.abilities.hint.ValueHint;
+import mage.abilities.keyword.FlyingAbility;
+import mage.abilities.keyword.IndestructibleAbility;
+import mage.cards.Card;
+import mage.cards.CardImpl;
+import mage.cards.CardSetInfo;
+import mage.constants.*;
+import mage.filter.common.FilterControlledPermanent;
+import mage.filter.common.FilterCreatureCard;
+import mage.filter.predicate.permanent.TokenPredicate;
+import mage.game.Game;
+import mage.game.permanent.Permanent;
+import mage.game.permanent.token.BloodToken;
+import mage.players.Player;
+import mage.target.TargetCard;
+import mage.target.common.TargetCardInHand;
+import mage.target.common.TargetControlledPermanent;
+import mage.target.targetpointer.FixedTarget;
+import mage.watchers.common.PlayerLostLifeWatcher;
+
+import java.util.*;
+
+/**
+ * @author Alex-Vasile
+ */
+public class StrefanMaurerProgenitor extends CardImpl {
+
+    private static final Hint hint = new ValueHint("Players who lost life this turn", StrefanMaurerProgenitorNumberPlayersLostLifeDynamicValue.instance);
+    private static final FilterControlledPermanent bloodTokenFilter = new FilterControlledPermanent("Blood tokens");
+
+    static {
+        bloodTokenFilter.add(SubType.BLOOD.getPredicate());
+        bloodTokenFilter.add(TokenPredicate.TRUE);
+    }
+
+    public StrefanMaurerProgenitor(UUID ownerId, CardSetInfo setInfo) {
+        super(ownerId, setInfo, new CardType[]{CardType.CREATURE}, "{2}{B}{R}");
+
+        this.supertype.add(SuperType.LEGENDARY);
+        this.subtype.add(SubType.VAMPIRE);
+        this.subtype.add(SubType.NOBLE);
+
+        this.power = new MageInt(3);
+        this.toughness = new MageInt(2);
+
+        // Flying
+        this.addAbility(FlyingAbility.getInstance());
+
+        // At the beginning of your end step, create a Blood token for each player who lost life this turn.
+        this.addAbility(
+                new BeginningOfEndStepTriggeredAbility(
+                        new CreateTokenEffect(
+                                new BloodToken(),
+                                StrefanMaurerProgenitorNumberPlayersLostLifeDynamicValue.instance),
+                        TargetController.YOU,
+                        false)
+                        .addHint(hint),
+                new PlayerLostLifeWatcher()
```

Comment:
No need to add the watcher.  It's one of the ones that gets added automatically.

https://github.com/magefree/mage/blob/4dd4ca12c1d5eab684162a74faeeb74b9d7fc9ce/Mage/src/main/java/mage/watchers/common/PlayerLostLifeWatcher.java#L13-L19